### DISCUSSION TITLE
How can I get traceId  except use MDC in log when use auto instrumentation

### DISCUSSION THREAD

**Discussion by jack161641** [open-telemetry/opentelemetry-java-instrumentation#4531] [Q&A]
How do I get the traceid because I want to put it in the return message body with the traceid when use auto instrumentation 
I try use MDC.get("span_id"),but I get null value,But I can see traceId in the log

**Comment by anuraaga** [comment]
Hi @jack161641 - you should be able to use `Span.current().getSpanContext().getTraceId()`. If you haven't already, you will need to add `io.opentelemetry:opentelemetry-api` as a dependency to your project.

**Reply by jack161641** [reply]
@anuraaga Thanks
